common/models/access-token.js

```diff
@@ -132,26 +137,26 @@ module.exports = function(AccessToken) {
    * @param {Boolean} isValid
    */
 
-  AccessToken.prototype.validate = function(cb) {
+  AccessToken.validate = function(token, cb) {
```

Comment:
@danilat pardon me if my answer is obvious, but why is it necessary to change this method from a prototype method to a static method?